security_module_twin/azureiotsecurity_default.json

```diff
@@ -2,29 +2,85 @@
   "moduleId": "azureiotsecurity",
   "properties": {
     "desired": {
-      "azureiot*com^securityAgentConfiguration^1*0*0": {
-        "highPriorityMessageFrequency": "PT7M",
-        "lowPriorityMessageFrequency": "PT5H",
-        "snapshotFrequency": "PT13H",
-        "maxLocalCacheSizeInBytes": 2560000,
-        "maxMessageSizeInBytes": 204800,
-        "eventPriorityConnectedHardware": "Low",
-        "eventPriorityListeningPorts":  "High",
-        "eventPriorityProcessCreate": "Low",
-        "eventPriorityProcessTerminate": "Low",
-        "eventPrioritySystemInformation": "Low",
-        "eventPriorityLocalUsers": "High",
-        "eventPriorityLogin": "High",
-        "eventPriorityConnectionCreate": "Low",
-        "eventPriorityFirewallConfiguration": "Low",
-        "eventPriorityOSBaseline": "Low",
-        "eventPriorityDiagnostic": "Low",
-        "eventPriorityConfigurationError": "Low",
-        "eventPriorityDroppedEventsStatistics": "Low",
-        "eventPriorityMessageStatistics": "Low"
+      "ms_iotn:urn_azureiot_Security_SecurityAgentConfiguration": {
```

Comment:
PnP has no versioning?

Reply:
The version exists on the interface id and not in the object name anymore